client/i18n/locales/chinese/trending.json

```
@@ -1,62 +1,62 @@
+  "article9title": "MongoDB Atlas 教程",
+  "article9link": "https://chinese.freecodecamp.org/news/get-started-with-mongodb-atlas/",
+  "article10title": "Python 字符串转数字",
+  "article10link": "https://chinese.freecodecamp.org/news/python-convert-string-to-int-how-to-cast-a-string-in-python/",
+  "article11title": "Git 命令",
+  "article11link": "https://chinese.freecodecamp.org/news/how-to-use-basic-git-and-github-commands/",
+  "article12title": "更新 NPM 依赖",
+  "article12link": "https://chinese.freecodecamp.org/news/how-to-update-npm-dependencies/",
+  "article13title": "谷歌恐龙游戏",
+  "article13link": "https://chinese.freecodecamp.org/news/do-you-know-the-chrome-dino-game-millions-of-people-are-playing/",
+  "article14title": "毛玻璃效果",
+  "article14link": "https://chinese.freecodecamp.org/news/glassmorphism-design-effect-with-html-css/",
+  "article15title": "Python 获取时间",
+  "article15link": "https://chinese.freecodecamp.org/news/how-to-get-the-current-time-in-python-with-datetime/",
+  "article16title": "Git Clone 指定分支",
+  "article16link": "https://chinese.freecodecamp.org/news/git-clone-branch-how-to-clone-a-specific-branch/",
+  "article17title": "JS 字符串反转",
+  "article17link": "https://chinese.freecodecamp.org/news/how-to-reverse-a-string-in-javascript-in-3-different-ways/",
+  "article18title": "React 个人作品网站",
+  "article18link": "https://chinese.freecodecamp.org/news/portfolio-app-using-react/",
+  "article19title": "媒体查询范围",
+  "article19link": "https://chinese.freecodecamp.org/news/media-queries-width-ranges/",
+  "article20title": "forEach 遍历数组",
+  "article20link": "https://chinese.freecodecamp.org/news/javascript-foreach-how-to-loop-through-an-array-in-js/",
+  "article21title": "撤销 Git Add",
+  "article21link": "https://chinese.freecodecamp.org/news/how-to-undo-a-git-add/",
+  "article22title": "OSI 七层网络",
+  "article22link": "https://chinese.freecodecamp.org/news/osi-model-networking-layers/",
+  "article23title": "Event Loop 执行顺序",
+  "article23link": "https://chinese.freecodecamp.org/news/node-js-event-loop/",
+  "article24title": "CMD 删除文件",
+  "article24link": "https://chinese.freecodecamp.org/news/cmd-delete-folder-how-to-remove-files-and-folders-in-windows/",
+  "article25title": "Git 删除分支",
+  "article25link": "https://chinese.freecodecamp.org/news/how-to-delete-a-git-branch-both-locally-and-remotely/",
+  "article26title": "HTML 表格代码",
+  "article26link": "https://chinese.freecodecamp.org/news/html-tables-table-tutorial-with-css-example-code/",
+  "article27title": "如何学习 Python",
+  "article27link": "https://chinese.freecodecamp.org/news/learning-python-from-zero-to-hero/",
+  "article28title": "REST API 教程",
+  "article28link": "https://chinese.freecodecamp.org/news/rest-api-tutorial-rest-client-rest-service-and-api-calls-explained-with-code-examples/",
+  "article29title": "C 语言入门",
+  "article29link": "https://chinese.freecodecamp.org/news/the-c-beginners-handbook/",
```

```suggestion
  "article29link": "https://chinese.freecodecamp.org/news/the-c-beginners-handbook/"
```

The lint issue Mrugesh was referencing, so that this isn't blocked.